h2/config.py

```diff
@@ -87,13 +93,18 @@ def __init__(self,
                  header_encoding='utf-8',
                  validate_outbound_headers=True,
                  normalize_outbound_headers=True,
-                 validate_inbound_headers=True):
+                 validate_inbound_headers=True,
+                 logger=None):
         self.client_side = client_side
         self.header_encoding = header_encoding
         self.validate_outbound_headers = validate_outbound_headers
         self.normalize_outbound_headers = normalize_outbound_headers
         self.validate_inbound_headers = validate_inbound_headers
 
+        default_logger = logging.getLogger(__name__)
```

Comment:
You mentioned NullLogger in #11 to avoid any conditionals, and I thought this is what you meant.

In terms of replacing this with a DummyLogger, would your approach just be to make a object that inherits from the logger, and makes all the logging level calls no-ops?